src/main/scala/Top.scala

```diff
@@ -0,0 +1,44 @@
+package flexpret
+import chisel3._
+import flexpret.core.{Core, FlexpretConfiguration, GPIO, HostIO}
+import flexpret.Wishbone.{HelloWishbone, WishboneMaster}
+
+case class TopConfig(
```

Comment:
How's this different from `FlexpretConfiguration`?

Reply:
This will be the config for the entire SoC. We need a Top level module where we can instantiate the core and also IO devices, NoC etc. This is that top level module